components/webext-storage/src/api.rs

```diff
@@ -332,6 +332,45 @@ pub fn get_bytes_in_use(conn: &Connection, ext_id: &str, keys: JsonValue) -> Res
     Ok(size)
 }
 
+/// Information about the usage of a single extension.
+#[derive(Debug, Clone, PartialEq)]
+pub struct UsageInfo {
+    /// The extension id.
+    pub extension: String,
```

Comment:
Femto-nit: Unless we specifically want to use `extension` in the Sync ping, would you mind spelling this `ext_id` or `extension_id`, to match other uses?